### PR TITLE
using CF tasks form running migrations

### PR DESCRIPTION
**Pull request by alexey-klimuk**
Using CF tasks for running db and data migrations during deploy

https://docs.cloudfoundry.org/devguide/using-tasks.html

### CODE REVIEW

**Comment by alexey-klimuk** [comment]
[[TARIFF16] Update deployment to use CF tasks](https://trello.com/c/pI6sWso2/1505-tariff16-update-deployment-to-use-cf-tasks)

File: bin/deploy [hunk 1]
```diff
@@ -23,8 +23,11 @@ cf create-app-manifest $CF_APP
 cf create-app-manifest $CF_APP_WORKER
 
 # Database migrations
-cf push "$CF_APP"-migrations -f "$CF_APP_WORKER"_manifest.yml -c "rake db:migrate" -i 1
-cf push "$CF_APP"-migrations -f "$CF_APP_WORKER"_manifest.yml -c "rake db:data:migrate" -i 1
+cf run-task $CF_APP "bundle exec rake db:migrate" --name "$CF_APP"-db-migrate
+cf run-task $CF_APP "bundle exec rake db:data:migrate" --name "$CF_APP"-db-data-migrate
+
+# cf push "$CF_APP"-migrations -f "$CF_APP_WORKER"_manifest.yml -c "rake db:migrate" -i 1
```
Comment (matthewford) [inline bin/deploy]: I would remove the comments

File: bin/deploy [hunk 2]
```diff
@@ -23,8 +23,11 @@ cf create-app-manifest $CF_APP
 cf create-app-manifest $CF_APP_WORKER
 
 # Database migrations
-cf push "$CF_APP"-migrations -f "$CF_APP_WORKER"_manifest.yml -c "rake db:migrate" -i 1
-cf push "$CF_APP"-migrations -f "$CF_APP_WORKER"_manifest.yml -c "rake db:data:migrate" -i 1
+cf run-task $CF_APP "bundle exec rake db:migrate" --name "$CF_APP"-db-migrate
+cf run-task $CF_APP "bundle exec rake db:data:migrate" --name "$CF_APP"-db-data-migrate
```
Comment (matthewford) [inline bin/deploy]: See my comment on the card, I'm not 100% sure with Sequel, but you need to run migrations after the deploy, and probably deploy again to reset the db schema cache.

**Comment by matthewford** [comment]
You know, now im looking at this - due to the two deploys (one to get the code for the db migrations and another to restart the app) im not sure this is much better than what we have now. closing for now

**Pull request closed without merging**